src/ScaleUnitManagement/UserConfig.sample.xml

```diff
@@ -3,26 +3,15 @@
   
```

Comment:
We should not add breaking changes to the configuration lightly. Users will all have to update their configurations and if we're not helping them migrate (with some tool or automatically) then it will cause confusion and frustration.

I'm not sure what the right approach is here, but I have some ideas:
* We could add a `<Version>2.0</Version>` element, look for it in the config and if it's not there: explain them the delta from v1 to v2 (when they run the tool) such that they know how and what to change.
* We could add (as described above) automatic migration, where we update the file if the `Version` element is not present and just add the `Version` element and populate `/CloudAndEdgeConfiguration/ScaleUnitConfiguration/ScaleUnitInstance/AuthConfiguration` with details from `/CloudAndEdgeConfiguration/AADConfiguration`. This way they are seamlessly upgraded without having to worry about it.
* (Least optimal, but easiest to execute): Add the `Version` element and document our way around it. This will probably lead to a bit of frustration as users need to go figure out themselves why things are failing and how to solve it (i.e. search in our documentation).

Reply:
Could we just release 3.0.0 version with the message that this contains major configuration changes so they should update UserConfig.xml according to new UserConfig.Sample.xml. And with that explain how to do the conversion and potentially provide automation for it.

If we want to avoid changes as big as this, then we can get back to my initial implementation with ADFS configuration being in the separate node. That should not break anyone as it is in special section not used by anyone who is not using LBD.

Reply:
@alkalomo what do you think?